release/smbchk.py

```diff
@@ -70,12 +79,20 @@ def nm(ns):
     #   -C: demangles the C++ symbols.
     #   -fbsd: formats output in default 'bsd' style
     # note that we are only going to pick up symbols in the cyclus namespace
-    # because everything else should be external to the cares of cyclus stability
+    # because everything else should be external to the cares of cyclus
+    # stability
     lib = os.path.abspath(os.path.join(ns.prefix, 'lib', 'libcyclus.so'))
     stdout = subprocess.check_output(['nm', '-g', '-p', '-C', '-fbsd', lib])
     names = set()
-    ok_types = {'B', 'b', 'D', 'd', 'R', 'r', 'S', 's', 'T', 't', 'W', 'w', 'u'}
+    ok_types = {'B', 'b', 'D', 'd', 'R', 'r',
+                'S', 's', 'T', 't', 'W', 'w', 'u'}
     for line in stdout.splitlines():
+        # replace c++11 symbol by standard one
+        for symbol in CPP11_SYMBOLS_REPLACEMENTS:
+            line = line.replace(symbol[0], symbol[1])
+        # fix string replacement when a '>' follows the string
+        line = line.replace("string >", "string>")
```

Comment:
I still think that there should only be a single replacements list that covers all replacements and that this pair should be included there.